### PR TITLE
Add init-buildx script for better log description

### PR DESCRIPTION
**Pull request by Prajyot-Parab**
**What this PR does / why we need it**:
- Add init-buildx script for better log description

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

/area provider/ibmcloud

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note
Add init-buildx script for better log description
```

### CODE REVIEW

File: hack/init-buildx.sh [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  exit 1
+fi
+
+# Ensure qemu is in binfmt_misc
+# Docker desktop already has these in versions recent enough to have buildx
+# We only need to do this setup on linux hosts
+if [ "$(uname)" == 'Linux' ]; then
+  # NOTE: this is pinned to a digest for a reason!
+  # Note2 (@rikatz) - Removing the pin, as apparently it's breaking new alpine builds
+  # docker run --rm --privileged multiarch/qemu-user-static@sha256:28ebe2e48220ae8fd5d04bb2c847293b24d7fbfad84f0b970246e0a4efd48ad6 --reset -p yes
```
Comment (mkumatag) [inline hack/init-buildx.sh:38]: do we really need this? commented lines doesn't make any sense!

File: hack/init-buildx.sh [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if [ -n "$DEBUG" ]; then
+	set -x
+fi
+
+# set -o errexit
+# set -o nounset
+# set -o pipefail
```
Comment (mkumatag) [inline hack/init-buildx.sh:23]: remove if not used

**Review by mkumatag — CHANGES_REQUESTED** [review]

File: hack/init-buildx.sh (hunk 1 above)
Comment (Prajyot-Parab) [reply hack/init-buildx.sh]: removed.

**Review by mkumatag — APPROVED** [review]
/lgtm

**Comment by mkumatag** [comment]
@Prajyot-Parab wondering if we can run this in a local environment to debug further via runpj?

**Pull request merged**